src/admin/class-media.php

```diff
@@ -8,36 +8,59 @@
 
 namespace Inf_Theme\Admin;
 
+use Eightshift_Libs\Core\Service;
+
 use Inf_Theme\Helpers\General_Helper;
 use Inf_Theme\Helpers\Object_Helper;
 
 /**
  * Class Media
+ *
+ * This class handles all media options. Sizes, Types, Features, etc.
  */
-class Media {
+class Media implements Service {
```

Comment:
I'd like to leave this, but if we'd want to publish a theme to wp.org, I'm 99% sure the theme wouldn't pass, because of the svg upload enabling (sounds like a plugin territory). So we'd need to document this properly for anyone who wants to use it for this purpose.

Reply:
let's agree this is going to be stupid boilerplate without forcing anything to the user.

All this featurs will be in our BP and WIKI